src/Messages/TSMessagesManager.h

```diff
@@ -20,14 +21,6 @@ NS_ASSUME_NONNULL_BEGIN
 @interface TSMessagesManager : NSObject
 
 - (instancetype)init NS_UNAVAILABLE;
-
-- (instancetype)initWithNetworkManager:(TSNetworkManager *)networkManager
```

Comment:
I'm not sure I understand precisely what you are proposing.

We can't omit the definition from production.

Though we're no longer using this declaration in production, we still use the definition in production, it's just private, and ultimately used to implement + sharedManager.